spec/searchmetrics_client/response_spec.rb

```diff
@@ -12,8 +12,9 @@
     include_context 'with api credentials'
 
     describe '#value' do
-      it 'return array data' do
-        expect(response.value).to be_instance_of Array
+      it 'return hash data' do
+        expect(response.value).to be_instance_of Hash
+        expect(response.value).to eq(balance: 59_981)
```

Comment:
Yes you are right this not correct response recorded on the `VCR` cassette, I will look it up.